conf/countries.json

```diff
@@ -1804,5 +1795,225 @@
     "official-name": "The Republic of Poland",
     "name": "Poland",
     "citizen-names": "Pole"
+  },
+  "AI" : {
+    "country" : "AI",
+    "name" : "Anguilla"
```

Comment:
We have extra spaces after Codes, countries and names. Its up to you if you want to format them

Reply:
I don't think it makes any difference in JSON. The data in mongo looks ok.